posts/2022-03-29-3-body-problem.md

```diff
@@ -0,0 +1,35 @@
+---
+_Is this series for you?_
+
+If you’re a CIO, CTO, LOB leader, cloud architect,  or professional developer, interested in the future of development and the relationship between public chains, private chains, cloud services, and conventional IT. Or if you are considering one more blockchain projects or trying to build a cost effective partner data sharing solution. 
+
+## Quick Overview
+
+Conventional (aka centralized) application approaches, which are increasingly based on cloud services, are often portrayed as antithetical to both private and public blockchains. 
+
+In reality, **all three categories are likely to emerge as key elements of modern IT solutions** because each possesses unique, durable advantages over the other two.
+
+*Emerging best-of-breed practices employ these technologies for their differentiated benefits:*
+
+- **Conventional (aka “centralized”) IT solutions**, increasingly built in the cloud and often on top of managed cloud services, will continue to offer the highest throughput, lowest latency, and easiest governance indefinitely.
+    
+    These will continue to form 80% or more of the foundational layer for SMB, mid-market, and enterprise IT.
+    
+- **Private/permissioned blockchains** have the advantage of innovating quickly, and can integrate seamlessly with the cloud solutions of their controlled populations, **making them ideal for creating partner buses that enable businesses to share data with rigorous compliance, privacy, and governance capabilities.** 
+**Conventional (“dumb pipe”) APIs will be replaced over time with distributed ledgers** that offer an automatically maintained and cryptographically provable single source of truth for cross-company and cross-org use cases such as supply chains, financial ledgers, and media/file sharing.
+    
+    Private chains will grow to represent 15-20% of IT stacks.
+    
+- **Public blockchains** offer very limited throughput due to the intrinsic requirements of Byzantine Attack resistance, planet-wide data replication, and “single box” deployment models that intentionally forgo public cloud leverage. Public chains will continue to exhibit high latency, limited data type support, and modest innovation rates (due to their open source/consortium nature). However, **they will play a key role for applications that require public disclosure, non-repudiation and/or total ordering among unrelated parties, or where public data markets exist,** serving key use cases such as the archival storage for NFT ownership, self-service ticket transfers among consumers, and third party auditing of (hashed, encrypted digests) of key financial and business data.
+    
+    Public chain integration will eventually comprise 1-3% of typical enterprise IT stacks over the long term.
+    
+In the upcoming series we will go over how the 3 key categories of tech work today, their strengths and weaknesses, and an example using Adobe Stock™️ that helps meet their business needs. 
```

Comment:
Might be helpful to describe the example a bit more. I don't think this reads well without context.